docs/pages/guides/google-authentication.mdx

```diff
@@ -40,72 +39,42 @@ We recommend uploading the app to the Google Play Store if your app still needs
   href="https://expo.fyi/first-android-submission"
 />
 
-Once you have uploaded your app, you can provide an SHA-1 certificate fingerprint value when asked while configuring the Google project. There are two values that you can provide from:
+### Configure your Firebase or Google Cloud Console project
+
+> Refer to the [package documentation](https://react-native-google-signin.github.io/docs/setting-up/get-config-file) for a more in-depth configuration guide.
+
+For Android, once you have uploaded your app, you need to provide the SHA-1 certificate fingerprint values when asked while configuring the project in Firebase or Google Cloud Console. There are two types of values that you can provide:
 
 - Fingerprint of the **.apk** you built (on your machine or using EAS Build). You can find the SHA-1 certificate fingerprint in the Google Play Console under **Release** > **Setup** > **App Integrity** > **Upload key certificate**.
-- Fingerprint of a **production app** downloaded from the play store. You can find the SHA-1 certificate fingerprint in the Google Play Console under **Release** > **Setup** > **App Integrity** > **App signing key certificate**.
+- Fingerprint(s) of a **production app** downloaded from the play store. You can find the SHA-1 certificate fingerprint(s) in the Google Play Console under **Release** > **Setup** > **App Integrity** > **App signing key certificate**.
 
 ### With Firebase
 
-For more instructions on how to configure your Google project for Android and iOS with Firebase:
+For more instructions on how to configure your project for Android and iOS with Firebase:
 
 <BoxLink
   title="Firebase"
-  href="https://react-native-google-signin.github.io/docs/setting-up/expo#expo-and-firebase"
+  href="https://react-native-google-signin.github.io/docs/setting-up/expo#expo-and-firebase-authentication"
 />
 
 #### Upload google-services.json and GoogleService-Info.plist to EAS
 
-If you use the Firebase method for Android and iOS (as shared in sections above), you'll need to upload **google-services.json** and **GoogleService-Info.plist** to EAS and add them to **.gitignore** to avoid checking them in the repository.
+If you use the Firebase method for Android and iOS (as shared in sections above), you'll need to make sure **google-services.json** and **GoogleService-Info.plist** are available in EAS for building the app. You can check them into your repository because the files should not contain sensitive values, or you can treat the files as secrets, add them to **.gitignore** and use the guide below to make them available in EAS.
 
 <BoxLink
   title="Upload a secret file to EAS and use in the app config"
   href="/build-reference/variables/#how-to-upload-a-secret-file-and-use-it-in-my-app-config"
```

Comment:
```suggestion
  href="/eas/environment-variables/?redirected#use-environment-variables-with-eas-build"
```

Let's update this link to the new EAS env variables guide since the current also redirects to this page.